optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -588,6 +590,7 @@ def _from_transformers(
             force_download=force_download,
             ov_config=ov_config,
             library_name=cls._library_name,
+            model_variant=variant,
```

Comment:
switching between `model_variant` and `variant` is confusing, we should probably stick to one, `model_variant`  or `weight_variant` should be fine as this more of a weights file specification

Reply:
yeah let's avoid `variant` and stick to `weights_variant` in this case

Reply:
but in this case, it will be unaligned with original from_pretrained argument
https://huggingface.co/docs/transformers/main_classes/model#transformers.PreTrainedModel.from_pretrained.variant

is it ok or better to keep as variant?

Reply:
I'll use weights_variant everywhere except from_pretrained now, could you please take a look?

Reply:
> but in this case, it will be unaligned with original from_pretrained argument
https://huggingface.co/docs/transformers/main_classes/model#transformers.PreTrainedModel.from_pretrained.variant
is it ok or better to keep as variant in from_pretrainedt?

`variant` makes more sense now since it's the argument the core libs are using especially because it's used in the model cards https://huggingface.co/Efficient-Large-Model/Sana_1600M_512px_MultiLing_diffusers#1-how-to-use-sanapipeline-with-%F0%9F%A7%A8diffusers. @echarlaix wdyt

Reply:
renamed